File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-independent.java
List<List<Item>> groups = loadGroups();
int groupIndex = 0;
while (groupIndex < groups.size()) {
    List<Item> items = groups.get(groupIndex);
    int itemIndex = 0;
    while (itemIndex < items.size()) {
        processItem(items.get(itemIndex));
        itemIndex = itemIndex + 1;
    }
    List<Notification> notifications = loadNotifications(groups.get(groupIndex));
    int notificationIndex = 0;
    while (notificationIndex < notifications.size()) {
        sendNotification(notifications.get(notificationIndex));
        notificationIndex = notificationIndex + 1;
    }
    groupIndex = groupIndex + 1;
}
showSummary();

File: packages/gunreip/laravel-translation-workbench/resources/views/pages/tw-graph/samples/documentation/idea-to-paper/flow/while/code-examples/while-independent.js
const groups = loadGroups();
let groupIndex = 0;
while (groupIndex < groups.length) {
    const items = groups[groupIndex];
    let itemIndex = 0;
    while (itemIndex < items.length) {
        processItem(items[itemIndex]);
        itemIndex = itemIndex + 1;
    }
    const notifications = loadNotifications(groups[groupIndex]);
    let notificationIndex = 0;
    while (notificationIndex < notifications.length) {
        sendNotification(notifications[notificationIndex]);
        notificationIndex = notificationIndex + 1;
    }
    groupIndex = groupIndex + 1;
}
showSummary();
